WordPress/gutenberg
# Pages data view: Update quick-actions

## What?
* Make 'Edit' a primary action so that it appears in the quick-action icon buttons.
* Swap the order so that 'Edit' is the first action, both in quick-actions and the ellipsis menu.

## Testing Instructions
1. Open the Pages data view.
2. Observe the Edit is is the first quick-action, and the first item in the ellipsis menu.

## Before
<img width="1457" alt="Screenshot 2024-03-14 at 09 52 23" src="https://github.com/WordPress/gutenberg/assets/846565/b3d85f36-5636-4e51-9040-c7b581e043e9">


## After

<img width="1299" alt="Screenshot 2024-03-14 at 15 00 21" src="https://github.com/WordPress/gutenberg/assets/846565/a5b3508d-0311-46f8-b6dc-ca8f8a2547c4">
